helm/conjur-app-deploy/charts/app-secrets-provider-rotation/values.yaml

```diff
@@ -0,0 +1,27 @@
+# Default values for the Application with Secrets Provider init container in Push-to-File mode Helm chart
```

Comment:
```suggestion
# Default values for the Application with Secrets Provider sidecar container in Push-to-File mode Helm chart
```

Reply:
Thanks for catching this! I'll fix and recommit